prac_06/guitar.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404 Prac_06 - guitar.py
+Mia Parnell
+Estimated time: 35 minutes
+Actual time: 40 minutes
+"""
+
+CURRENT_YEAR = 2023
+
+
+class Guitar:
```

Comment:
Just missing a docstring here

Reply:
I've actually missed one here I think? 
Would you consider using a constant to track how old a guitar is to be considered vintage?